src/components/BaseStories/Heading.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from "react"
+import {
+  Box,
+  Flex,
+  Heading as HeadingComponent,
+  HeadingProps,
+  Stack,
+  VStack,
+} from "@chakra-ui/react"
+import { Meta, StoryObj } from "@storybook/react"
+import Translation from "../Translation"
+
+type HeadingType = typeof HeadingComponent
+
+const meta = {
+  title: "Atoms / Typography / Heading",
+  component: HeadingComponent,
+  parameters: {
+    layout: null,
+  },
+  decorators: [
+    (Story) => (
+      <Flex align="center" minH="100vh">
+        <Story />
+      </Flex>
+    ),
+  ],
+} satisfies Meta<HeadingType>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+const headingScale: Array<HeadingProps> = [
+  {
+    as: "h1",
```

Comment:
@TylerAPfledderer @pettinarip maybe this is normal, but having 3 times  `as="h1"` is correct?

Reply:
@nloureiro I could only assume that the three largest sizes would be reserved for the `h1` heading. If that is true, then I would make sure that the story reflected that preference.